yt/frontends/stream/tests/test_stream_particles.py

```diff
@@ -456,3 +456,73 @@ def test_stream_sph_projection():
     image = frb["gas", "density"]
     assert image.max() > 0
     assert image.shape == (256, 256)
+
+
+def test_stream_non_cartesian_particles_unigrid():
```

Comment:
note for the future: code duplication in these tests could be dealt with in one pytest fixture
Maybe add this as an inline comment for future-us ?
```suggestion
@pytest.mark.parametrize("loader", (load_uniform_grid, load_amr_grid))
def test_stream_non_cartesian_particles(loader):
```